readme.md

```diff
@@ -29,22 +29,22 @@ This can be achieved by:
 
 And both pin/lock to the same version.
 
-# npm@<7
+# Yarn
 
-`npm@<7` does not automatically install `peerDependencies`,
+Yarn does not automatically install `peerDependencies`,
 so if that's what you're using, install them manually.
 Here is an example, but use it only for reference,
 because your decisions regarding version ranges and range specifiers may vary.
 
-```
-npm install --save-dev \
+```sh
+yarn add \
```

Comment:
Is there no equivalent to `--save-dev`?

Reply:
There is, I've added it in a second commit after this one.